src/testdrive/src/action/postgres.rs

```diff
@@ -7,8 +7,23 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+use tokio_postgres::{Client, NoTls};
+
+mod connect;
 mod execute;
 mod verify_slot;
 
+pub use connect::build_connect;
 pub use execute::build_execute;
 pub use verify_slot::build_verify_slot;
+
+pub async fn postgres_client(url: &String) -> Result<Client, String> {
+    let (client, connection) = tokio_postgres::connect(url, NoTls)
+        .await
+        .map_err(|e| format!("connecting to postgres: {}", e))?;
+
+    println!("Connecting to PostgreSQL server at {}...", url);
+    tokio::spawn(connection);
+
+    Ok(client)
+}
```

Comment:
This should go in `testdrive::util::postgres` as `client`.

Reply:
Moved, thanks.